cmd/ci-operator/main.go

```diff
@@ -465,6 +469,7 @@ func (o *options) Run() error {
 		}
 		if err != nil {
 			if !o.dry {
+				o.reportToSentry(err)
```

Comment:
I was expecting to see this in the outermost scope. Don't we miss important failures by reporting only `steps.Run`?

Reply:
Not that many, but you are right - I got lured there by an already existing dry run conditional block (I think we only want to report in real runs). I'll file a followup.